controllers/operator_controller.go

```diff
@@ -175,6 +175,17 @@ func (r *OperatorReconciler) reconcile(ctx context.Context, op *operatorsv1alpha
 		return ctrl.Result{}, err
 	}
 
+	if isBundleDepStale(existingTypedBundleDeployment) {
+		apimeta.SetStatusCondition(&op.Status.Conditions, metav1.Condition{
+			Type:               operatorsv1alpha1.TypeReady,
+			Status:             metav1.ConditionUnknown,
+			Reason:             operatorsv1alpha1.ReasonInstallationStatusUnknown,
```

Comment:
@joelanford (continuing the [discussion](https://github.com/operator-framework/operator-controller/issues/127#issuecomment-1449127217) here) - I am leaning towards leaving the status Unknown. If we for sure know that status is out of date because of an error where it can't anymore change to true, we could make it `False`. My understanding is:

`True` - We know for sure that installation was successful.
`Unknown` - Can't determine the status because either we don't have enough data (BD conditions aren't created), or rukpak is still in progress.
`False` - We know for sure that the package is not available (probably neither will it be in future given the current state persists).

 Which is why I was wondering if we need Rukpak to have a new status for this particular scenario of upgrades, rather than us inferring an out of date bd status. Till we reach to a point of introducing any more status conditions (addressing  #126), could we make this scenario to be "Unknown". 

I'm fine with making it "False" too, but just wanted to check again based on what we discussed [here](https://github.com/operator-framework/operator-controller/pull/130#discussion_r1114626377).

Reply:
+1 on unknown for now.